docs/source/mayavi/mlab_running_scripts.rst

```diff
@@ -37,31 +36,34 @@ Using together with Matplotlib's pylab
 If you want to use Matplotlib's pylab with Mayavi's mlab in IPython you
 should:
 
-    * if your IPython version is greater than 0.11: start IPython with::
+    * start IPython with::
 
-        $ ipython --gui=wx --pylab=wx
+        $ ipython --gui=qt --pylab=qt
```

Comment:
FYI: Since IPython 1.0.0 there is a `--matplotlib` switch that automatically select the backend (can be enforced with `--matplotlib=qt`).  `--pylab` is kept there for historical reason (so it would still work)